config/initializers/delayed_job.rb

```diff
@@ -3,3 +3,9 @@
 Delayed::Worker.max_run_time = 1.day
 Delayed::Worker.raise_signal_exceptions = :term
 Delayed::Worker.logger = Logger.new(Rails.root.join("log/delayed_job_#{Rails.env}.log"))
+Delayed::Worker.queue_attributes = {
+  transcodes: { priority: 5 },
```

Comment:
Shouldn't there be a queue for `AudioFile.check_file_attributes` ? It is meant to be ran as a delayed job

Reply:
It was only used in the migration, right? If we ever need to run that again it will probably be from the console (where choosing the priority manually is probably for the better). Only if we ever expose it via the API should a queue be added here, IMO.

Reply:
Yeah, sure